docs/api.md

```diff
@@ -0,0 +1,34 @@
+# ssh_scan API documentation
+
+**Last Updated:** 2016-09-19 <br>
+**Author:** agaurav77@yahoo.com <br>
+**API Version:** 1
+
+This document explains the ssh_scan Web API, intended to be used in the Web API for ssh_scan.
+
+## Protocol Calls
+
+### scan
+
+**API Call:** `scan` <br>
+**API Method:** `POST`
```

Comment:
Yeah, there would be GETs. Technically, GETs are for retrievals. Imagine if we start caching the results as well, then a GET could retrieve cached results and a POST could do a new scan.